pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -8,37 +8,43 @@ import org.folio.models.parameters.CreateNamespaceParameters
 
 //TODO Switch branch before final merge
 //@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
-@Library('pipelines-shared-library@RANCHER-1001') _
+@Library('pipelines-shared-library@RANCHER-1247') _
 
 properties([
 //  pipelineTriggers([cron('H 6,9,12,15,18,21 * * 1-5')]),
-buildDiscarder(logRotator(numToKeepStr: '40')),
-disableConcurrentBuilds(),
-parameters([
-  string(name: 'FOLIO_BRANCH', defaultValue: 'snapshot', description: 'platform-complete branch where will be run quality CI job'),
-  booleanParam(name: 'SKIP_CHECK', defaultValue: false, description: 'Skip check of changes since last build.'),
-  folioParameters.agent(),
-  folioParameters.refreshParameters()
-])
+  buildDiscarder(logRotator(numToKeepStr: '40')),
+  disableConcurrentBuilds(),
+  parameters([
+    string(name: 'FOLIO_BRANCH', defaultValue: 'snapshot', description: 'platform-complete branch where will be run quality CI job'),
+    booleanParam(name: 'SKIP_CHECK', defaultValue: false, description: 'Skip check of changes since last build.'),
+    folioParameters.agent(),
+    folioParameters.refreshParameters()])
 ])
 
 if (params.REFRESH_PARAMETERS) {
   currentBuild.result = 'ABORTED'
   return
 }
 
+Logger logger = new Logger(this, env.JOB_BASE_NAME)
+
 boolean diffFound = false
 boolean autoCanceled = false
-Logger logger = new Logger(this, env.JOB_BASE_NAME)
-String clusterName = 'folio-tmp' //TODO Switch to folio-testing cluster before final merge
+
+//TODO switch to regular job before merge
+String createNamespaceFromBranchJob = '/folioRancher/tmpFolderForDraftPipelines/createNamespaceFromBranch-1247'
+//TODO switch to regular job before merge
+String deleteNamespaceJob = '/folioRancher/tmpFolderForDraftPipelines/deleteNamespace-1247'
```

Comment:
Are we considering moving pipeline paths (maybe to Constants) so they aren't written as strings in dependent pipelines?
Maybe some class or module to work with dependent pipelines.

Reply:
yeap, we have, it will be separate PR